config/routes.rb

```diff
@@ -401,14 +401,18 @@
           # sometimes underscores get messed up when linked to via SMS
           as: :capture_doc_dashes
 
-      get '/in_person_proofing/state_id' => 'in_person/state_id#show'
+      #Deprecated route - temporary redirect while state id changes are rolled out
```

Comment:
I checked your build and the lint step is failing on this line.  There needs to be a space after the above #.